csharp/src/Apache.Arrow/Types/DecimalType.cs

```diff
@@ -17,6 +17,7 @@ namespace Apache.Arrow.Types
 {
     public sealed class DecimalType: FixedSizeBinaryType
     {
+        public static readonly DecimalType Default = new DecimalType(0, 0);
```

Comment:
I'm not sure a decimal with precision = 0 is valid.

Looking at the C++ implementation, I don't see a "default" DecimalType being defined. Maybe it is best to leave it up to the caller?